migrations/versions/330_add_lot_table_and_relationships.py

```diff
@@ -0,0 +1,97 @@
+"""Add lot table and relationships
+
+Revision ID: 330
+Revises: 320_drop_selection_answers
+Create Date: 2015-10-14 10:48:29.311523
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '330'
+down_revision = '320_drop_selection_answers'
+
+import itertools
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+
+
+def upgrade():
+    # Create Lots table and Lot to Framework relationship
+    op.create_table(
+        'lots',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('slug', sa.String(), nullable=False),
+        sa.Column('name', sa.String(), nullable=False),
+        sa.Column('one_service_limit', sa.Boolean(), nullable=False),
+        sa.PrimaryKeyConstraint('id')
+    )
+
+    op.create_index(op.f('ix_lots_slug'), 'lots', ['slug'], unique=False)
+
+    op.create_table(
+        'framework_lots',
+        sa.Column('framework_id', sa.Integer(), nullable=True),
```

Comment:
I don't think either column in this table should be nullable - you need both IDs to record a relationship, an entry with just one ID is meaningless.